.github/workflows/test_and_validate.yml

```diff
@@ -223,17 +217,17 @@ jobs:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - uses: actions/checkout@v3
 
-      - name: Build Submodules
-        id: submod_build_1
-        uses: ./.github/actions/ngen-submod-build
-        with: 
-          mod-dir: "extern/test_bmi_fortran/"
-      
-      - name: Build Submodules
-        id: submod_build_2
-        uses: ./.github/actions/ngen-submod-build
-        with: 
-          mod-dir: "extern/iso_c_fortran_bmi/"
+      # - name: Build Submodules
+      #   id: submod_build_1
+      #   uses: ./.github/actions/ngen-submod-build
+      #   with: 
+      #     mod-dir: "extern/test_bmi_fortran/"
+      # 
+      # - name: Build Submodules
+      #   id: submod_build_2
+      #   uses: ./.github/actions/ngen-submod-build
+      #   with: 
+      #     mod-dir: "extern/iso_c_fortran_bmi/"
```

Comment:
Commented code can presumably now be deleted?

Reply:
Yes that is correct, I'll make that adjustment in a bit -- thanks for pointing that out